tests/metrics_test.go

```diff
@@ -153,14 +158,95 @@ var _ = Describe("Metrics", func() {
 		)
 	})
 
+	Context("SSP metrics", func() {
+		getDeployment := func(name string, namespace string) *apps.Deployment {
```

Comment:
This function is only used in one place, so it can be inlined.

Reply:
using it twice after changes